app/controllers/users_controller.rb

```diff
@@ -1,5 +1,10 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.all
+  end
 
-  def show; end
+  def show
+    @user = User.find(params[:id])
+    @recent_posts = @user.three_recent_posts
```

Comment:
[OPTIONAL] Instead of defining a new instance variable in the controller `@recent_posts`, you can use `three_recent_posts` method directly in the view:
```rb
<% @user.three_recent_posts.each_with_index do |post,index|  %>
```

Reply:
Hello @medaminedev66 , It was an error merging the PR. Sorry for that